src/components/MediaItem/MediaItemTitle.tsx

```diff
@@ -75,7 +77,9 @@ export default function MediaItemTitle({ i, data, type, parentRef }: Props) {
   return (
     <h3 className='mb-2 flex sm:text-xl'>
       <span className='mr-1.5 inline-flex items-baseline gap-1' ref={numberRef}>
-        <span className='font-bold text-black'>#{i + 1} </span>
+        <span className={`font-bold ${top3Colors[i] ?? 'text-white'}`}>
```

Comment:
Nothing wrong with this, but since we use `clsx` for concatenating across the project, I would like to keep it consistent. Also would use `||` to check for falsy values instead.

```
<span className={clsx('font-bold', top3Colors[i] || 'text-white')}>
  #{i + 1}{' '}
</span>
```

Reply:
I Haven't heard of `clsx` before, thanks.
Is there any reason you prefer `||` over `??`. I personally would use `??`, since an empty string would be accepted and not defaulted back. This will probably never happen tho., so this isn't really a big deal.

I am just intrested to learn the reasoning.

Reply:
I guess my thought process was that any falsy value might lead to unexpected styling where no classes are applied and it would be better to just fallback to `text-white` in all of these cases, specifically an empty string.

But yeah, I do agree that it's really a non-issue in this instance.